statefun-examples/statefun-python-walkthrough/README.md

```diff
@@ -0,0 +1,46 @@
+# Apache Stateful Functions - Walkthrough
+
+## Setup
+
+* Create a virtual env
+
+```
+python3 -m venv venv
+source venv/bin/activate   
+```
+
+* Install the requirements 
+
+```
+pip3 install requirements.txt
+```
+
+If you are building from source, then first build the 
+distribution (via calling `statefun-python-sdk/build-distribution.sh`)
+then copy `statefun-python-sdk/dist/apache_flink_statefun-1.1_SNAPSHOT-py3-none-any.whl` here and
```

Comment:
```suggestion
then copy `statefun-python-sdk/dist/apache_flink_statefun-2.1_SNAPSHOT-py3-none-any.whl` here and
```

or maybe `apache_flink_statefun-*-py3-none-any.whl` so that its oblivious to the branch version